spec/person_spec.rb

```diff
@@ -0,0 +1,22 @@
+# spec/person_spec.rb
+require_relative '../person'
+require_relative '../rental'
+
+describe Person do
+  let(:person) { Person.new(20, 'John Doe') }
+
+  describe '#initialize' do
+    it 'creates a new person with the given age and name' do
+      expect(person.age).to eq(20)
+      expect(person.name).to eq('John Doe')
+    end
+
+    it 'generates a random ID for the person' do
+      expect(person.id).to be_an(Integer)
+    end
+
+    it 'initializes rentals as an empty array' do
+      expect(person.rentals).to be_empty
+    end
+  end
```

Comment:
- [ ] Dear @BHObserver, Please keep in mind we need to add unit tests for all the methods in all the classes. I noticed you have failed to test the `can_use_services`, `correct_name`, `add_rental`, and `of_age` methods inside the person class. Please add tests that will assert these methods are working accordingly.